doc/Support/Cleanup-options.md

```diff
@@ -14,12 +14,15 @@ These options rely on ```daily.sh``` running from cron as per the installation i
 Cleanup Options are set in ```config.php```
 
 ```php
-$config['syslog_purge']                              = 30;
 $config['eventlog_purge']                            = 30;
+$config['syslog_purge']                              = 30;
+$config['route_purge']                               = 10;
+$config['alert_log_purge']                           = 365;
 $config['authlog_purge']                             = 30;
+$config['ports_fdb_purge']                           = 10;
 $config['device_perf_purge']                         = 7;
-$config['rrd_purge']                                 = 90;// Not set by default
```

Comment:
Why did you change these two to remove the notes that they aren't enabled by default?

Reply:
I think the inconsistency between the default config and the document is misleading. It's clearly the units are days, not necessary to have `90` as an example. If `90` is a suggested value, I think we need to change the default config instead of the document.

For `ports_purge`, it's set to `10` instead of `true` or `false` by default. The original value and comment are misleading.

\* Although I have traced the related code, I am not 100% sure about how it works. Please let me know if something wrong/misunderstanding.

Reply:
@Sea-n You could also add that these parameters can be configured directly and graphically in the Web GUI instead of the old way in config.php.

Reply:
@PipoCanaja Thanks for the suggestion! I will add that later.